requirements.txt

```diff
@@ -0,0 +1,9 @@
+h5py>=2.10.0
```

Comment:
Just realised, we probably don't need this either. But let's just merge anyway!

Reply:
You're right, I should have checked again. I'll change it in my next PR!